src/actions/signUpAction.js

```diff
@@ -0,0 +1,19 @@
+import { SIGNUP } from "./types";
+
+const REGISTER_URL = "https://ah-backend-thor.herokuapp.com/api/users/";
```

Comment:
Could you put the link in an env file so that you donot explicitly put it in the action